src/Features/Core/Portable/Completion/Providers/AbstractObjectCreationCompletionProvider.cs

```diff
@@ -79,48 +80,79 @@ protected override Task<ImmutableArray<ISymbol>> GetSymbolsWorker(SyntaxContext
 
             if (type.SpecialType == SpecialType.System_Void)
             {
-                return SpecializedTasks.EmptyImmutableArray<ISymbol>();
+                return ImmutableArray<ISymbol>.Empty;
             }
 
             if (type.ContainsAnonymousType())
             {
-                return SpecializedTasks.EmptyImmutableArray<ISymbol>();
+                return ImmutableArray<ISymbol>.Empty;
             }
 
             if (!type.CanBeReferencedByName)
             {
-                return SpecializedTasks.EmptyImmutableArray<ISymbol>();
+                return ImmutableArray<ISymbol>.Empty;
+            }
+
+            if (type.IsInterfaceType())
+            {
+                var typeArity = type.GetArity();
+
+                var implementations = await SymbolFinder.FindImplementationsAsync(
+                    type, context.Workspace.CurrentSolution, cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
Yup, I'm leaving this for now, I want to delay this as long as I can and focus on the basic functionality first, I'll get to it, eventually. ;)